ydb/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+TRIES=3
+set -e
+source benchmark_variables.sh
+
+YDB_PASSWORD=password
+
+cert_dir=$(find ydb-ansible-examples/TLS/CA/certs -maxdepth 1 -type d -not -path "." -printf "%T@ %p\n" | sort -n | tail -n 1 | cut -d' ' -f2-)
+
+cat queries.sql | while read -r query; do
+    echo -n "["
```

Comment:
It would be good to drop the filesystem cache at the beginning of each query to make sure "cold" runs are really cold. Grep `/proc/sys/vm/drop_caches` in the repository for precedence.

Reply:
YDB uses raw block devices, that means we do not have any filesystem caches at all. But I can add it if it is required

Reply:
Oh wow. In that case, there is no need to add anything.
Perhaps only add a comment (basically what you wrote ^^).